src/main/scala/dissonance/model/Timestamp.scala

```diff
@@ -0,0 +1,11 @@
+package dissonance.model
+
+import io.circe.{Decoder, Encoder}
+import java.time.Instant
+
+case class Timestamp(instant: Instant)
```

Comment:
Is the API set up to have another level or why do we have the Timestamp case class?

Reply:
I originally thought that `Instant` had a decoder based on the Unix epoch, which is what we need, but it doesn't. So I replaced everywhere that `Instant` was with this `Timestamp` and it decodes and encodes using the epoch. Another approach would be to add an implicit `Encoder[Instant]` and `Decoder[Instant]` and import those everywhere we need it, but I think this is safer. Thoughts?

Reply:
I don't love that we have `Timestamp` and `Timestamps`... I wonder if we want to make it like `EpochTimestamp` or something? idk. Either way, I think this is good

Reply:
I think I'm going to leave it to keep them the same as the names in the Discord docs, and I think it makes sense since `Timestamps` is just a pair of `Timestamp`s.